setup.py

```diff
@@ -35,6 +36,35 @@ def __str__(self):
             language='c++'
         )
     )
+elif platform.system() == 'Darwin':
+    ext_modules.append(
+        Extension('pyvirtualcam._native_macos',
+            # Sort input source files to ensure bit-for-bit reproducible builds
+            # (https://github.com/pybind/python_example/pull/53)
+            sorted([
+                'pyvirtualcam/native_macos/main.mm',
+                'pyvirtualcam/native_macos/OBSDALMachServer.mm',
+                'pyvirtualcam/native_macos/yuv_rgb.cpp']),
+            include_dirs=[
+                # Path to pybind11 headers
+                get_pybind_include(),
+                'pyvirtualcam/native_macos'
+            ],
+            extra_link_args=[
+                "-framework", "AVFoundation",
+                "-framework", "AppKit",
+                "-framework", "Cocoa",
+                "-framework", "CoreFoundation",
+                "-framework", "CoreMedia",
+                "-framework", "CoreVideo",
+                "-framework", "Cocoa",
+                "-framework", "CoreMediaIO",
+                "-framework", "IOSurface",
+                "-framework", "IOKit"
+            ],
+            language='objc'
```

Comment:
I have no idea if this is a valid option

Reply:
Yes, according to https://docs.python.org/3/distutils/apiref.html it is.